schemas/catalog.json

```diff
@@ -32,7 +32,8 @@
       "$ref": "#/definitions/description"
     },
     "links": {
-      "$ref": "#/definitions/links"
+      "$ref": "#/definitions/links",
+      "propertyOrder": 1007
```

Comment:
The validator works on draft-07 of JSON Schema, which doesn't support other properties alongside $ref. That's fine for UI annotations, but for other annotations it should be allOfs, otherwise other annotations would be ignored. Just fyi.

Reply:
As long as it is ignored by the validation and doesn't cause trouble, I guess it's fine then?